### ISSUE TITLE
Change defaults for format-style printing?

### ISSUE THREAD

**Issue by skirpichev** [aleaxit/gmpy#506]
Documentation [says](https://gmpy2.readthedocs.io/en/latest/mpfr.html#gmpy2.mpfr.__format__): The default format is '.6f'.  While this can be considered as a feature, I would prefer to be compatible with CPython floats here (in short, in CPython default is almost same as 'g').

With 'f' it's easy to demolish significand to dust:
```pycon
>>> f'{gmpy2.mpfr(1.3335678912e-10)}'
'0.000000'
>>> f'{gmpy2.mpfr(1.3335678912e-10):g}'
'1.33357e-10'
```

**Comment by casevh** [comment]
I agree that matching CPython is important. I don't remember why I chose '.6f'.

**Comment by skirpichev** [comment]
@casevh, same spammer as in #590.
